get_forecast.py

```diff
@@ -324,60 +324,57 @@ def calc_forecast(boto3_session):
     return output
 
 
-def format_rows(output,account_width):
-    #print the heading
-    mtd_width=8
-    forecast_width=8
-    change_width=6
+def format_rows(output, account_width):
+    # print the heading
+    forecast_width = 8
+    change_width = 8
 
-    output_rows=[]
+    total_forecast = 0
+    total_change = 0
+    
+    output_rows = []
 
-    row = {
-        "Account": 'Account'.ljust(account_width),
-        "MTD": 'MTD'.rjust(mtd_width),
-        "Forecast": 'Forecast'.rjust(forecast_width),
-        "Change": 'Change'.rjust(change_width)
+    # add new row for column headings
+    row_headings = {
+        "Account": "Account".ljust(account_width),
+        "Forecast": "Forecast".ljust(forecast_width),
+        "Change": "Change".ljust(change_width)
     }
-    output_rows.append(row)
+    
+    output_rows.append(row_headings)
 
-    #print in decending order by forecast
+    # print in descending order by forecast
     lines = sorted(output, key=lambda k: k.get('amount_forecast'), reverse=True)
-    for line in lines :
+    for line in lines:
         if len(lines) == 2 and line.get('account_name') == 'Total':
             continue
         change = "{0:,.1f}%".format(line.get('forecast_variance'))
         row = {
             "Account": line.get('account_name')[:account_width].ljust(account_width),
-            "MTD": "${0:,.0f}".format(line.get('amount_usage')).rjust(mtd_width),
```

Comment:
MTD shoul dnot ve removed from the row, if you look at how the code is written it is configurable, all you had to dos was suppress it on. line 361 and maybe improve the construction at the end but not sure